war/src/test/js/widgets/config/mocks.js

```diff
@@ -1,17 +1,18 @@
+// TODO: remove or do something about this when reenabling the unit tests
```

Comment:
Is this comment still valid?

Reply:
Not anymore :D good catch, I'll remove it